### ISSUE TITLE
Qtile window manager

### ISSUE THREAD

**Issue by GoodBoyNeon** [catppuccin/catppuccin#1709]
**Are you available to help in the creation of this port? 😄 (optional)**
Probably

**Additional context 📷 (optional)**
It's just the bar that requires some work. The rest like the window border can be ported from the corresponding color palettes.

**Comment by ghostx31** [comment]
What exactly do you mean by a qtile port? 

Do you mean something like our [catppuccin/hyprland](https://github.com/catppuccin/hyprland) port where you will define a python file for each flavour with all the colours and the users will import this file to implement the colours? Or do you mean something else?

**Comment by GoodBoyNeon** [comment]
> What exactly do you mean by a qtile port?
> 
> Do you mean something like our [catppuccin/hyprland](https://github.com/catppuccin/hyprland) port where you will define a python file for each flavour with all the colours and the users will import this file to implement the colours? Or do you mean something else?
> 
> Would it be cool if you can add your repository to showcase your idea probably?

Yes, a standalone qtile configuration file, very similar to [catppuccin/hyprland](https://github.com/catppuccin/hyprland) but it won't include stuff like wallpaper engine or any third-party software. 
It really should include just the bar configuration and border colors. As it's not applicable for a theme to have keymappings, scripts, etc. configured.

**Comment by ghostx31** [comment]
Sure that is a good idea. Looking forward to your contribution!

**Comment by GoodBoyNeon** [comment]
This is how my desktop currently looks:
![image](https://user-images.githubusercontent.com/93624576/197949049-23388a67-b646-4275-8616-d8a9bd60003f.png)
![2022-10-26_12-01](https://user-images.githubusercontent.com/93624576/197949193-a7713dcb-73e5-46c6-b3d8-594619bb14f7.png)

The bar definitely needs some more widgets and customization. This is just a starting point!
